public/gameList.php

```diff
@@ -64,6 +65,7 @@ function ListGames(
         echo "<th class='text-right'>Achievements</th>";
         echo "<th class='text-right'>Points</th>";
         echo "<th class='text-right'>Retro Ratio</th>";
+        echo "<th style='white-space: nowrap'>Last Updated</th>";
```

Comment:
This should be right aligned too. - Developer Sets page
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/5c5ef37c-fa69-4fe2-be7b-8a597cf63834)

Reply:
Thanks, resolved in https://github.com/RetroAchievements/RAWeb/pull/1841/commits/94d47dc42d0352a238a68f2ca3f1ca58ffb97bbc.